docs/contributor/04-10-technical-design.md

```diff
@@ -1,22 +1,22 @@
 # Technical Design
 
-## Kyma Istio operator
+## Kyma Istio Operator
 
-![Kyma IstioOperator Overview](./istio-operator-overview.svg)
+![Kyma IstioOperator Overview](./../assets/istio-operator-overview.svg)
 
 We want to keep the Kyma Istio operator as simple as possible. That's why we decided to start with one controller that consists of several self-contained components
```

Comment:
[Our guidelines are against "we"](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md)

Reply:
The whole doc is written from the team's perspective. If that's not a problem, I'd rather merge this PR first, and work on the structure of this doc later when the content is up-to-date (here's the [issue](https://github.com/kyma-project/istio/issues/188) for updating it).